src/main/java/step3/domain/LottoInfo.java

```diff
@@ -0,0 +1,24 @@
+package step3.domain;
+
+public class LottoInfo {
+    private static final int chargeOfLotto = 1000;
```

Comment:
```suggestion
    private static final int CHARGE_OF_LOTTO = 1000;
```

상수는 대문자로 작성해주는 것이 관례입니다!

Reply:
수정하였습니다!